src/strategies/math/index.ts

```diff
@@ -12,36 +52,155 @@ export async function strategy(
   options,
   snapshot
 ): Promise<Record<string, number>> {
-  const upstreamResult: Record<string, number> = await strategies[
-    options.strategy.name
-  ].strategy(
-    space,
-    options.strategy.network,
-    getProvider(options.strategy.network),
-    addresses,
-    options.strategy.params,
-    snapshot
+  const strategyOptions: Options = migrateLegacyOptions(options);
```

Comment:
I feel it is better to create a new strategy than change the existing options, it will confuse a lot of users

Reply:
I understand the concern. However, the `math` strategy was created for composibility, and would be greatly limited in terms of what can be built on top of it if multi-operand support cannot be added.

Also I just checked and there doesn't seem to be any space using this strategy yet:

```sh
$ curl "https://hub.snapshot.org/api/explore" | jq ".strategies.math"
null
```